src/main/java/competition/subsystems/turret/TurretSubsystem.java

```diff
@@ -0,0 +1,62 @@
+package competition.subsystems.turret;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import competition.IdealElectricalContract;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+
+@Singleton
+public class TurretSubsystem extends BaseSubsystem {
+    
+    public XCANTalon motor;
+    DoubleProperty maxAngleProp;
+    DoubleProperty minAngleProp;
+    DoubleProperty turnSpeedProp;
+
+    @Inject
+    public TurretSubsystem(CommonLibFactory factory, PropertyFactory pf, IdealElectricalContract contract) {
+        log.info("Creating TurretSubsystem");
+        pf.setPrefix(this);
+        
+        maxAngleProp = pf.createPersistentProperty("MaxAngle",180);
+        minAngleProp = pf.createPersistentProperty("MinAngle", -180);
+        turnSpeedProp = pf.createPersistentProperty("TurnSpeed", .03); 
+
+
+        this.motor = factory.createCANTalon(contract.rotationMotor().channel);
+        motor.setInverted(contract.rotationMotor().inverted);
+    }
+
+    public void TurnLeft()
+    {
+        motor.simpleSet(-turnSpeedProp.get());
```

Comment:
In the world of the unit circle, positive power == positive rotation == counter-clockwise rotation == turning to the left. This should be +.